vignettes/getting-started.Rmd

```diff
@@ -74,6 +92,18 @@ You'll notice that this set up gives us a list of formatted data. This contains
 str(gimap_dataset)
 ```
 
+```{r}
+gimap_dataset <- run_qc(gimap_dataset, params = list(cell_line = "pgPEN_HeLa"), overwrite = TRUE)
+```
+
+The addition to this is that now a QC filter has been added to the `gimap_dataset` object, specifically found in `gimap_dataset$transformed_data$qc_filter`. It's a list of TRUE and FALSE. With TRUE meaning that it passes the filters and should be kept, and FALSE meaning that it didn't pass (right now the combined filter) and should not be kept for the analysis. Future work should let you select if you're using one, both, or neither of the filters.
+
+```{r}
+gimap_dataset$transformed_data$qc_filter
+```
+
+Future work will also produce an HTML report for all of the plots and tables made while running the QC process.
```

Comment:
Future work?

Reply:
It was future work at the time I made that comment. I forgot to remove it on Friday night.

Reply:
Fixed in commit [fa2e9df](https://github.com/FredHutch/gimap/pull/16/commits/fa2e9df149923fd54cb84627d17448040e3bd2e4)